xarray/backends/api.py

```diff
@@ -185,12 +185,28 @@ def _finalize_store(write, store):
     store.close()
 
 
+def load_dataset(*args, **kwargs):
+    """Open, load into memory, and close a dataset from a file or file-like
+    object.
+    """
```

Comment:
It would be good to document that arguments are passed on to `open_dataset`, and include a reference to `open_dataset` under a "See also" section. That gets turned into a link in the online API docs.

Also: there are probably a couple of arguments that should be invalid with `load_dataset`, e.g., `cache` is meaningless if you're loading the data into memory anyways.

Reply:
The docstring should also spell out exactly how this differs from `open_dataset`.